src/lib/request.ts

```diff
@@ -53,9 +28,55 @@ type PromisifiedRequest = typeof requestLib & {
 	) => Bluebird<any>;
 };
 
-const requestHandle = requestLib.defaults(exports.requestOpts);
+const getRequestInstances = once(async () => {
+	// Generate the user agents with out versions
```

Comment:
```suggestion
	// Generate the user agents with our versions
```
otherwise, it does not make much sense...